### PR TITLE
Create css-aural

### PR DESCRIPTION
**Pull request by brennanyoung**
First step towards mapping support for CSS 3 Speech Module is to create this stub which records the fact that CSS 2.1 Aural has been deprecated / superseded by the newer spec.

### CODE REVIEW

**Comment by brennanyoung** [comment]
I wasn't sure about all those "n" stats. The table should be almost empty - a stub.

File: features-json/css-aural [hunk 1]
```diff
@@ -0,0 +1,330 @@
...
+      "url":"http://lab.dotjay.com/notes/css/aural-speech/",
+      "title":"Aural CSS: Support for CSS 2 Aural Style Sheets / CSS 3 Speech Module"
+    },
+    {
+      "url":"https://css-tricks.com/lets-talk-speech-css/",
+      "title":"Let’s Talk About Speech CSS"
+    }
+  ],
+  "bugs":[
+    {
```
Comment (cvrebert) [inline features-json/css-aural]: Instead of this dummy entry, you should just set `bugs` to an empty list.

**Comment by Fyrd** [comment]
In these cases I prefer to make search results instead of full feature data. So I've taken the information from the PR and added it here:
https://caniuse.com/#search=aural

Thanks!

**Comment by Malvoz** [comment]
Can we add spec: https://drafts.csswg.org/css-speech-1/ as suggested in [this comment](https://github.com/Fyrd/caniuse/issues/1240#issuecomment-350392853)?

**Pull request closed without merging**